<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountEditorDialog</class>
 <widget class="QDialog" name="AccountEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Account Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/tray1.png</normaloff>images/tray1.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Friendly name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_FriendlyName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_LoginName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server domain:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Domain">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Port">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>993</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Secure">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Secure SSL</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Save">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch program:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Command">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you set an URL, don't forget &quot;http://&quot;, otherwise it won't work!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Browse">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>269</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gmail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GMail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_outlook">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Outlook.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use preset:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_FriendlyName</tabstop>
  <tabstop>line_LoginName</tabstop>
  <tabstop>line_Password</tabstop>
  <tabstop>line_Domain</tabstop>
  <tabstop>line_Port</tabstop>
  <tabstop>check_Secure</tabstop>
  <tabstop>line_Command</tabstop>
  <tabstop>button_Browse</tabstop>
  <tabstop>button_Save</tabstop>
  <tabstop>button_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
